test_runner/fixtures/zenith_fixtures.py

```diff
@@ -453,67 +453,62 @@ def pg_bin(test_output_dir: str, pg_distrib_dir: str) -> PgBin:
     return PgBin(test_output_dir, pg_distrib_dir)
 
 
-def read_pid(path):
-    """ Read content of file into number """
-    return int(Path(path).read_text())
-
-
 class WalAcceptor:
     """ An object representing a running wal acceptor daemon. """
-    def __init__(self, wa_binpath, data_dir, port, num):
+    def __init__(self, wa_binpath: str, data_dir: str, port: int, num: int):
         self.wa_binpath = wa_binpath
         self.data_dir = data_dir
         self.port = port
         self.num = num  # identifier for logging
+        self.process: Optional[Popen[bytes]] = None
 
     def start(self) -> 'WalAcceptor':
         # create data directory if not exists
         Path(self.data_dir).mkdir(parents=True, exist_ok=True)
 
-        cmd = [self.wa_binpath]
-        cmd.extend(["-D", self.data_dir])
-        cmd.extend(["-l", "localhost:{}".format(self.port)])
-        cmd.append("--daemonize")
-        cmd.append("--no-sync")
-        # Tell page server it can receive WAL from this WAL safekeeper
-        cmd.extend(["--pageserver", "localhost:{}".format(DEFAULT_PAGESERVER_PORT)])
-        cmd.extend(["--recall", "1 second"])
+        cmd = [
```

Comment:
IMHO the previous style was more readable, with the --option and its argument on the same line.

Reply:
Indeed. This is what I've come up with:

```python
cmd = [
    self.wa_binpath,
    *['--dir', self.data_dir],
    *['--listen', f'localhost:{self.port}'],
    # Tell page server it can receive WAL from this WAL safekeeper
    *['--pageserver', f'localhost:{DEFAULT_PAGESERVER_PORT}'],
    *['--recall', '1 second'],
    '--no-sync',
]
```